libraries/entities/src/DiffTraversal.cpp

```diff
@@ -125,36 +119,14 @@ void DiffTraversal::Waypoint::getNextVisibleElementDifferential(DiffTraversal::V
                 if (nextElement) {
                     AACube cube = nextElement->getAACube();
-                    if (view.viewFrustum.calculateCubeKeyholeIntersection(cube) != ViewFrustum::OUTSIDE) {
-                        // check for LOD truncation
-                        float renderAccuracy = calculateRenderAccuracy(view.viewFrustum.getPosition(),
-                                                                       cube,
-                                                                       view.rootSizeScale,
-                                                                       view.lodLevelOffset);
-
-                        if (renderAccuracy > 0.0f) {
-                            ViewFrustum::intersection lastIntersection = lastView.viewFrustum.calculateCubeKeyholeIntersection(cube);
-                            if (lastIntersection != ViewFrustum::INSIDE || nextElement->getLastChanged() > lastView.startTime) {
-                                next.element = nextElement;
-                                // NOTE: for differential case next.intersection is against the lastView
-                                // because this helps the "external scan" optimize its culling
-                                next.intersection = lastIntersection;
-                                return;
-                            } else {
-                                // check for LOD truncation in the last traversal because
-                                // we may need to traverse this element after all if the lastView skipped it for LOD
-                                renderAccuracy = calculateRenderAccuracy(lastView.viewFrustum.getPosition(),
-                                                                         cube,
-                                                                         lastView.rootSizeScale,
-                                                                         lastView.lodLevelOffset);
-
-                                if (renderAccuracy <= 0.0f) {
-                                    next.element = nextElement;
-                                    // element's intersection with lastView was effectively OUTSIDE
-                                    next.intersection = ViewFrustum::OUTSIDE;
-                                    return;
-                                }
-                            }
+                    // check for LOD truncation
+                    float distance = glm::distance(view.viewFrustum.getPosition(), cube.calcCenter()) + MIN_VISIBLE_DISTANCE;
+                    float apparentAngle = cube.getScale() / distance;
+                    if (apparentAngle > MIN_ELEMENT_APPARENT_ANGLE * view.lodScaleFactor) {
+                        if (view.viewFrustum.calculateCubeKeyholeIntersection(cube) != ViewFrustum::OUTSIDE) {
+                            next.element = nextElement;
+                            next.intersection = ViewFrustum::OUTSIDE;
+                            return;
```

Comment:
The `Differential` case gets much simpler because we must scan all visible elements for entities that may have changed LOD culling between the last completed view traversal and the current one.